examples/java/src/test/java/org/apache/beam/examples/complete/TfIdfIT.java

```diff
@@ -52,7 +52,7 @@
 
   @BeforeClass
   public static void setUp() {
-    PipelineOptionsFactory.register(TfIdfITOptions.class);
+    PipelineOptionsFactory.register(TestPipelineOptions.class);
```

Comment:
I don't think you need this change after you added the extra project to the configuration.

Reply:
Was wrong, see above comment.